index.html

```diff
@@ -133,15 +133,15 @@ <h4>Suzaan clarke</h4>
   <div class="container">
       <div class="title"><h2>Contact Us</h2></div>
       <form>
-          <input type="text" id="name" class="field" placeholder="Your Name">
-          <input type="text" id="email" class="field" placeholder="Your Email">
-          <textarea id="message" placeholder="Your Message" class="field"></textarea>
+          <input type="text" class="field" placeholder="Your Name">
+          <input type="text"  class="field" placeholder="Your Email">
+          <textarea  placeholder="Your Message" class="field"></textarea>
           <button class="submitBtn">Submit</button>
```

Comment:
and same here, this what i want you to change in my previous review,

Reply:
Done